vm.js

```diff
@@ -1429,6 +1434,8 @@ Object.subclass('Squeak.Object',
                 oops = this.decodeWords(numLits+1, this.bits, littleEndian);
             this.pointers = this.decodePointers(numLits+1, oops, oopMap); //header+lits
             this.bytes = this.decodeBytes(nWords-(numLits+1), this.bits, numLits+1, this.format & 3);
+            this.compiled = false;
+            this.ic = new Array(this.bytes.byteLength);
```

Comment:
I'd prefer if you initialize `ic` to `null` here. Otherwise we're allocating a ton of Arrays that are never going to be used.